### PR TITLE
Pc 28997 data home refacto home funnel conversion

### PR DESCRIPTION
**Pull request by MatthieuRipollPassCulture**
# DA PR

## Describe your changes

In this PR, we are adding two new models: `mrt_native__daily_user_home_module` and `mrt_native__daily_home_module_conversion`.

These two models are not optimized because we need to put them into production quickly. During the migration of other Firebase models, we will create intermediate models to refactor the code from certain subqueries.

Tag a reviewer if necessacy  @LucileRainteau @MatthieuRipollPassCulture  

## Jira ticket number and/or notion link

JIRA-ticket_number

### Type of change
- [ ] Fix (non-breaking change which corrects expected behavior)
- [ ] New fields (non-breaking change)
- [ ] New table (non-breaking change)
- [ ] Concept change (potentially breaking change which modifies fields according to new or evolving business concepts) 
- [ ] Table deletion (potentially breaking change which adds functionality/ table)
      
### Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] Fields have been snake_cased
- [ ] I have checked my modifications don't break downstream models
- [ ] If my changes concern incremental table, I have altered their schema to accomodate with field's creation/deletion
- [ ] I have made corresponding changes to the [tables documentation](https://www.notion.so/passcultureapp/Documentation-Tables-175a397a8e854ff4a55ae4f3620dbe3b)
- [ ] I have made corresponding changes to the [fields glossary](https://www.notion.so/passcultureapp/854a436a8f1541e1b6ec2a65f8bab600?v=798024ba90404b139e5a17407a3bc604)
- [ ] I have updated the dag in cases of dependencies
- [ ] My code passes CI/CD tests
- [ ] I will post on slack review channel and ensure to specify the duration of the review task: short (<10min), medium (<30min), long (>30min)

### CODE REVIEW

File: orchestration/dags/dependencies/import_analytics/sql/analytics/new_home_funnel_conversion.sql [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+  , booking_id
+  , module_displayed_date
+  , module_displayed_timestamp
+  , module_clicked_timestamp
+  , consult_venue_timestamp
+  , consult_offer_timestamp
+  , fav_timestamp
+  , booking_timestamp
+FROM displayed
+LEFT JOIN clicked
```
Comment (LucileRainteau) [inline orchestration/dags/dependencies/import_analytics/sql/analytics/new_home_funnel_conversion.sql:166]: tu te souviens pourquoi on avait cette sous-requete clicked ? quelle est la différence avec consultations ?
Je vois que ça nous permet de récupérer le click type. Mais est-ce qu'on peut pas récupérer cette info via le champs consultations.origin ?

File: orchestration/dags/dependencies/import_analytics/sql/analytics/new_home_funnel_conversion.sql [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+LEFT JOIN favorites
+  ON favorites.unique_session_id = displayed.unique_session_id
+  AND favorites.module_id = displayed.module_id
+LEFT JOIN `analytics_prod.firebase_bookings` as bookings
+  ON displayed.unique_session_id = bookings.unique_session_id
+  AND consultations.offer_id = bookings.offer_id
+  AND consultations.consult_offer_timestamp <= bookings.booking_timestamp
+)
+
+SELECT
```
Comment (cdelabre) [inline orchestration/dags/dependencies/import_analytics/sql/analytics/new_home_funnel_conversion.sql:187]: rajouter le call_id

File: orchestration/dags/dependencies/import_analytics/sql/analytics/new_home_funnel_conversion.sql [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+    , COUNT(DISTINCT CASE WHEN click_type = 'ConsultVideo' THEN unique_session_id ELSE NULL END) AS nb_sesh_consult_video
+    , COUNT( CASE WHEN consult_offer_timestamp IS NOT NULL THEN 1 ELSE NULL END) AS nb_consult_offer
+    , COUNT( CASE WHEN fav_timestamp IS NOT NULL THEN 1 ELSE NULL END) AS nb_fav
+    , COUNT(DISTINCT CASE WHEN booking_timestamp IS NOT NULL THEN unique_session_id ELSE NULL END) AS nb_sesh_booking
+    , COUNT( CASE WHEN booking_timestamp IS NOT NULL THEN 1 ELSE NULL END) AS nb_bookings
+    , COUNT( CASE WHEN booking_id IS NOT NULL THEN 1 ELSE NULL END) AS nb_bookings_non_cancelled
+    , SUM(delta_diversification) AS total_diversification
+  FROM final_raw
+  --LEFT JOIN `clean_prod.applicative_database_user` USING(user_id)
+LEFT JOIN `analytics_prod.diversification_booking`  USING(booking_id)
```
Comment (cdelabre) [inline orchestration/dags/dependencies/import_analytics/sql/analytics/new_home_funnel_conversion.sql:209]: Possible de faire la table en incrémentale et faire une vue par la suite pour jointure diversification_booking ? (histoire de prendre en compte les annulation)

**Review by cdelabre — COMMENTED** [review]
Questions; 
L'attribution d'une vue avec une réservation (ou autre se fait) sur quelle temporalité ? 
Est-ce que l'on ne veut pas le faire au niveau d'un item_id par ailleurs ?

**Review by carolinemestre-passculture — DISMISSED** [review]

**Review by carolinemestre-passculture — APPROVED** [review]

**Pull request merged**